modules/deploy/pages/deployment-option/cloud/serverless.adoc

```diff
@@ -0,0 +1,154 @@
+* `JoinGroup`                   
+* `Heartbeat`                   
+* `LeaveGroup`                  
+* `SyncGroup`                   
+* `DescribeGroups`              
+* `ListGroups`                  
+* `SASLHandshake`               
+* `ApiVersions`                 
+* `CreateTopics`                
+* `DeleteTopics`                
+* `DeleteRecords`               
+* `InitProducerID`              
+* `OffsetForLeaderEpoch`        
+* `AddPartitionsToTxn`          
+* `AddOffsetsToTxn`             
+* `EndTxn`                      
+* `TxnOffsetCommit`             
+* `DescribeACLs`                
+* `CreateACLs`                  
+* `DeleteACLs`                  
+* `DescribeConfigs`             
+* `AlterConfigs`                
+* `AlterReplicaLogDirs`         
+* `DescribeLogDirs`             
+* `SASLAuthenticate`            
+* `CreatePartitions`            
+* `DeleteGroups`                
+* `IncrementalAlterConfigs`
+
+== Unsupported features
+
+The following features are not yet supported: 
+
+* Redpanda Admin, HTTP Proxy, and Schema Registry APIs are not yet available. You can use an external schema registry.
+* Managed connectors
+* Data transforms
+
+To use these features, to run production workloads with consistently high throughput, or for greater control over your deployment and cluster settings, use Redpanda BYOC or Dedicated clusters. 
+
+== Serverless configurations
+
+Redpanda Serverless is opinionated about Kafka configurations, including the following:
+
+* Automatic topic creation is disabled. 
++
+Some systems expect the Kafka service to automatically create topics when a message is produced to a topic that doesn't exist. Serverless is opinionated about disabling automatic topic creation. Create topics in the Redpanda Cloud UI or with `rpk topic create`.
+
+* Cleanup policy is delete.
++
+Serverless currently doesn't support topic compaction. All Serverless topics have `cleanup.policy = delete`. 
+
+* Uses infinite retention to avoid cleaning up older keys.
++
+Frameworks like Kafka Streams and Kafka Connect enforce compaction for the changelog topics. Thus, Serverless can't be used with Kafka Streams (for stateful jobs) or Kafka Connect.
+
+== Beta credits and limits
+
+Redpanda issues 100 free credits for the Beta trial. Usage is based on the following:
+
+* Ingress: 0.1 credit per GB in (writes)
+* Egress: 0.1 credit per GB out (reads)
+* Partitions: 0.003 credit (metered in 10 min fractions) per partition/hr	
+* Storage: 0.1 credit (metered in 10 min fractions) per GB stored/mth	
+
+After either the 14 day trial time expires or the 100 credits expire, the cluster moves into a suspended state, and you won't be able to access your data in the Redpanda Cloud UI or with the Kafka API. There is a 14 day grace period following the end of the trial, in case you want to extend it. After that, the data is permanently deleted. 
+
+Each Beta trial can have five Serverless clusters. Each cluster has the following limits:
```

Comment:
Surface Serverless limits as a separate section. E.g. detach it from the free trial.